### ISSUE TITLE
Query Parameters

### ISSUE THREAD

**Issue by averagehat** [DCGenomics/seqr#42]
I remember some discussion of default query parameters we should use for querying. I remember there was one in particular that was almost required for Seqr but that information seems to be lost. I want to make sure results/sensitivity are correct to compare with blastx.

**Comment by lianyi** [comment]
Right, we'd need to fill that in. What's the current parameter being used ( I don't have a computer to access until later today ) ?

Simply & DRY

-----Original Message-----

**Comment by averagehat** [comment]
Right now it's just "sequence:<sequence>" and nothing else

**Comment by lianyi** [comment]
Perhaps we can add mm ( minimum should match) = 50% for now . The smaller value will include more distant homologous but possible more false positives.

We can also add another flavor : tanimoto score ( will send in another update on this soon), that might better rank the hits.

Simply & DRY

-----Original Message-----
